pkg/api/v1alpha1/cluster_types.go

```diff
@@ -566,6 +567,20 @@ const (
 	Kube124 KubernetesVersion = "1.24"
 )
 
+func GreaterOrEqual(kube1, kube2 KubernetesVersion) (bool, error) {
+	kube1Float, err := strconv.ParseFloat(string(kube1), 32)
```

Comment:
Can we use the semver package for this instead of parsing as a float?

Reply:
The "1.24" value we assign to Kube versions does not identify as valid semver strings.
The valid strings start with a `v`.

`Package semver implements comparison of semantic version strings. In this package, semantic version strings must begin with a leading "v", as in "v1.0.0".`

Reply:
Does our semver package have that restriction?
you can do `"1.20" + ".0"` to make it a full semver btw